includes/connection/class-wc-terms.php

```diff
@@ -49,12 +49,12 @@ public static function register_connections() {
 										'toType'        => $tax_object->graphql_single_name,
 										'fromFieldName' => $tax_object->graphql_plural_name,
 										'resolve'       => function( $source, array $args, AppContext $context, ResolveInfo $info ) use ( $tax_object ) {
-											$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $tax_object->name );											
-                                            
-											// Get the term ids that are associated with this $source
-											$terms = wp_list_pluck( get_the_terms( $source->ID, $tax_object->name ), 'term_id' );
-											
-											$resolver->set_query_arg( 'term_taxonomy_id', ! empty( $terms ) ? $terms : array( '0' ) );
+											$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $tax_object->name );
+
+
+											$term_ids = \wc_get_object_terms( $object_id, $taxonomy, 'term_id' );
```

Comment:
My bad :sweat_smile:, it should be `$tax_object->name` not `$taxonomy`.

Reply:
😆 yup, I saw that after too. It's corrected now.